### DISCUSSION TITLE
Initial Configuration Assistant appears every time i close SEB on iPad

### DISCUSSION THREAD

**Discussion by vlutsyuk-amplify** [SafeExamBrowser/seb-mac#346] [Q&A]
Hello Folks,

Firstly, I want to express my appreciation for your product. SEB is truly awesome.

The following issue applies to iPad devices. I'm using the App Self-Lock (AAC) Single App Mode to run some web assessments (not Moodle) in secure mode on the device. From a regular browser, the user clicks on a special URL that responds with a *.seb file, and SEB automatically starts, switching the user's device into ACC. When the assessment is finished, the user either presses the SEB quit button and types the "quit password", or programmatically quits through the quit URL. However, the Initial Configuration Assistant then appears on the screen, 
<img width="567" alt="Screenshot 2023-12-04 at 7 16 15 PM" src="https://github.com/SafeExamBrowser/seb-mac/assets/15686610/6efcd446-013c-4021-8830-91daf6a9c306">  
what is slightly annoying as the user needs to manually swipe it up to quit it.

is there a way not to show Initial Configuration Assistant after quitting?

**Comment by danschlet** [comment]
iOS apps never terminate themselves, at least that's what Apple's instructions to developers are (iOS terminates apps automatically when needing RAM). You can recognize professional iOS users by not having the unnecessary need to swipe up apps to terminate them...  

When SEB's client settings are not configured, then current versions of SEB for Windows and Mac display the https://safeexambrowser.org/start page with similar instructions (but less configuration options) as the Initial Configuration Assistant (ICA, will also come to Mac/Windows at some point). You can prevent the ICA being displayed by using a client configuration with the Start URL set to another URL than https://safeexambrowser.org/start (also not to an empty string). You could for example load a simple web page showing a logo of your institution and/or specific instructions for your students. 

To change the client configuration of SEB clients, you can use a sebs:// link or QR code sebs:// link to a config saved "for configuring clients" (or the new Config URL/QR Code containing a full config), sending a .seb file containing a client config, MDM or the Automatic Client Configuration option (which only requires entering the domain of an institution in the field "Enter the URL of an institution which supports SEB" or "Search Network" if students are in the institutions WiFi network or VPN).
